### PR TITLE
chore: remove unused ts-node dependencies

### PR DESCRIPTION
**Pull request by blumamir**
## Which problem is this PR solving?

- Following discussion [here](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/712), with the great eye of @Rauno56 , It was noticed that ts-node dependency is installed but not used in `instrumentation-aws-sdk` and `browser-extension-autoinjection`.

## Short description of the changes

- This PR removes these unused dependencies which can also slightly improve install and CI times.

When this PR merges, there will be no dependencies on `ts-node` in the repo

### CODE REVIEW

File: plugins/node/opentelemetry-instrumentation-aws-sdk/package.json [hunk 1]
```diff
@@ -1,83 +1,82 @@
 {
-  "name": "@opentelemetry/instrumentation-aws-sdk",
```
Comment (dyladan) [inline plugins/node/opentelemetry-instrumentation-aws-sdk/package.json:2]: This whole file appears to be retabbed with 4 spaces instead of 2

**Review by Rauno56 — APPROVED** [review]

File: plugins/node/opentelemetry-instrumentation-aws-sdk/package.json (hunk 1 above)
Comment (blumamir) [reply plugins/node/opentelemetry-instrumentation-aws-sdk/package.json:2]: Right, my IDE is automatically formatting my files. I'll fix that. Thanks for noticing

Comment (blumamir) [reply plugins/node/opentelemetry-instrumentation-aws-sdk/package.json]: Fixed

**Review by Flarna — APPROVED** [review]

**Review by dyladan — APPROVED** [review]

File: packages/opentelemetry-browser-extension-autoinjection/package.json [hunk 2]
```diff
@@ -84,4 +83,4 @@
     "react": "17.0.2",
     "react-dom": "17.0.2"
   }
-}
+}
```
Comment (dyladan) [inline packages/opentelemetry-browser-extension-autoinjection/package.json:86]: missing trailing newline

**Pull request merged**